scripts/nbconverted/high_weight_genes.py

```diff
@@ -0,0 +1,355 @@
+#   - Type D nodes can also be split into two types depending on direction of the high weight tail
+#   - Use the opposite high weight gene procedure however to reduce false positive but still capture some signal in the opposite high weight tail.
+#   
+# We apply the correction and visualize the adjusted distributions. A pathway coverage analysis will determine if these adjustments improve pathway discovery.
+
+# In[1]:
+
+import os
+import numpy as np
+import pandas as pd
+from sklearn import preprocessing
+
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+from tybalt.genes import high_weight_genes
+
+
+# In[2]:
+
+get_ipython().magic('matplotlib inline')
+plt.style.use('seaborn-notebook')
+
+
+# In[3]:
+
+sns.set(style='white', color_codes=True)
+sns.set_context('paper', rc={'font.size':14, 'axes.titlesize':20, 'axes.labelsize':16})   
+
+
+# In[4]:
+
+# High weight gene metrics
+metric = 'stddev'
+threshold = 2.5
+
+# Color dictionary
+algorithm_color_dict = {'PCA': '#a6cee3',
+                        'ICA': '#1f78b4',
+                        'NMF': '#b2df8a',
+                        'ADAGE': '#33a02c',
+                        'Tybalt': '#fb9a99',
+                        'VAE_100': '#e31a1c',
+                        'VAE_300': '#fdbf6f'}
+
+
+# In[5]:
+
+# Feature file locations
+pca_file = '../pancan_viz/data/pca_feature_rnaseq.tsv.gz'
+ica_file = '../pancan_viz/data/ica_feature_rnaseq.tsv.gz'
+nmf_file = '../pancan_viz/data/nmf_feature_rnaseq.tsv.gz'
+adage_file = 'https://github.com/greenelab/tybalt/raw/4bb7c5c5eb6b9dfe843269f8c3059e1168542b55/results/adage_gene_weights.tsv'
+tybalt_file = 'https://github.com/greenelab/tybalt/raw/928804ffd3bb3f9d5559796b2221500c303ed92c/results/tybalt_gene_weights.tsv'
+vae_2l100_file = 'https://github.com/greenelab/tybalt/raw/7d2854172b57efc4b92ca80d3ec86dfbbc3e4325/data/tybalt_gene_weights_twohidden100.tsv'
+vae_2l300_file = 'https://github.com/greenelab/tybalt/raw/7d2854172b57efc4b92ca80d3ec86dfbbc3e4325/data/tybalt_gene_weights_twohidden300.tsv'
+
+
+# In[6]:
+
+# Instantiate high weight gene objects
+pca_weights = high_weight_genes(pca_file, metric=metric, thresh=threshold, algorithm='PCA')
+ica_weights = high_weight_genes(ica_file, metric=metric, thresh=threshold, algorithm='ICA')
+nmf_weights = high_weight_genes(nmf_file, metric=metric, thresh=threshold, algorithm='NMF')
+adage_weights = high_weight_genes(adage_file, metric=metric, thresh=threshold, algorithm='ADAGE')
+tybalt_weights = high_weight_genes(tybalt_file, metric=metric, thresh=threshold, algorithm='Tybalt')
+vae_2l100_weights = high_weight_genes(vae_2l100_file, metric=metric, thresh=threshold, algorithm='VAE_100')
+vae_2l300_weights = high_weight_genes(vae_2l300_file, metric=metric, thresh=threshold, algorithm='VAE_300')
+
+
+# ## Visualize distributions
+# 
+# Certain `high_weight_genes` methods enable quick visualizations of high weigh gene features
+# 
+# 1. Positive and negative tail number of high weight genes
```

Comment:
This tails scatter plot is using the standard method right? I would remind the reader of that since you talk about the dynamic procedure above and I found myself digging into `count_high_weight_genes` to find out.

Reply:
sounds good, added in next commit :+1: